### ISSUE TITLE
Frontend tests sometimes fail: Expected to find element: '[data-test="signupBtn"]'

### ISSUE THREAD

**Issue by taoeffect** [okTurtles/group-income#706]
### Problem

Sometimes frontend tests will fail:

```
  1) SignUp, Profile and Login sign up new user1:
     CypressError: Timed out retrying: Expected to find element: '[data-test="signupBtn"]', but never found it.


  [... snip ...]

  (Results)

  ┌────────────────────────────────────────┐
  │ Tests:        9                        │
  │ Passing:      1                        │
  │ Failing:      1                        │
  │ Pending:      0                        │
  │ Skipped:      7                        │
  │ Screenshots:  1                        │
  │ Video:        false                    │
  │ Duration:     6 seconds                │
  │ Spec Ran:     signup-and-login.spec.js │
  └────────────────────────────────────────┘

  (Screenshots)

  - group-income-simple/test/cypress/screenshots/signup-and-login.spec.js/SignUp, Profile and Login -- sign up new user1 (failed).png (1280x720)
```

The screenshot it saves is:

![SignUp, Profile and Login -- sign up new user1 (failed)](https://user-images.githubusercontent.com/138706/66942735-5ecd2d00-f052-11e9-904f-d29cd327b8bc.png)

### Solution

Fix the test :)

**Comment by sandrina-p** [comment]
This isn't very useful @taoeffect. It should be explicit what steps have been done previously exactly. Until then, this doesn't tell us much about this "unstable" test suit.

**Comment by taoeffect** [comment]
@sandrina-p I'm not sure I follow. This is simply what happens when you run `grunt test`, same thing happens on Travis. For example, it's happening right now with @SebinSong's PR: https://travis-ci.org/okTurtles/group-income-simple/builds/599002753?utm_source=github_status&utm_medium=notification

**Comment by sandrina-p** [comment]
After reading around, it seems Cypress might be faster than Vue re-rendering. So, cypress might start the next set of tests before the logout process is fully finished (but that doesn't make much sense because the logout command asserts if the text "welcome to group income" exists and that passes 🤔)

Anyway, my attempt to fix this is to double-check the user is really logged out before trying to signup/login. I pushed that fix directly into PR #704. Let's wait a few runs/days to see if this fails again. If not, I think we can close this issue :)

**Comment by taoeffect** [comment]
Yeah this is still a serious issue, as Travis is failing most of the time now, and with different errors.

Here's a recent one: https://travis-ci.org/okTurtles/group-income-simple/builds/599248499

```
  6 passing (60s)
  1 failing
  1) Proposals - Add members user2 proposes to add user6 to the group:
     CypressError: Timed out retrying: Expected to find element: '[data-test="nextBtn"]', but never found it.
```

If you visit that link and it's green, it's because I just restarted it (and usually that fixes it).
